src/Sparrow/UnmanagedPointer.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Runtime.CompilerServices;
```

Comment:
Unnecessary include?

Reply:
Yes, leftover of both `UnmanagedPointer<T>` (which requires `CompilerServices`) and `UnmanagedPointer` living in the same project.